plugins/promela/hu.bme.mit.gamma.xsts.promela.transformation/src/hu/bme/mit/gamma/xsts/promela/transformation/serializer/ModelSerializer.xtend

```diff
@@ -268,8 +268,8 @@ class ModelSerializer {
 	'''
 	
 	protected def String serializeAsTrivialBranch(Action action) '''
-		:: true -> atomic {
-			«action.serialize»
+		:: «action.serialize» -> atomic {
```

Comment:
This solution makes sense only if the action is the instance of `AssumeAction`, right? So shouldn't we introduce a branching here?

```
if (action intanceof AssumeAction) {
    // Introduced code
}
else {
    // Deleted code
}
```

Reply:
Thank you, you are right, added previous code as else branch.